get_tracking_numbers.py

```diff
@@ -40,15 +40,15 @@ def main():
   print("Retrieving Amazon tracking numbers from email ...")
   amazon_tracking_retriever = AmazonTrackingRetriever(config, args, driver_creator)
   try:
-    trackings = amazon_tracking_retriever.get_trackings()
+    trackings = amazon_tracking_retriever.get_trackings(True)
```

Comment:
Avoid having necessary boolean parameters whenever possible; it's not clear at a glance what this parameter might mean and it limits us to only two possibilities.